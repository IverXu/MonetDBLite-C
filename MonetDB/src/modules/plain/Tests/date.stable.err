stderr of test 'date` in directory 'modules/plain` itself:

#> Mtest:	other date
#> Mtest:	./date date
#> Mlog:	Mtimeout -timeout 600 Mserver -yaccsilent -d8 -db modules_plain date.mil
