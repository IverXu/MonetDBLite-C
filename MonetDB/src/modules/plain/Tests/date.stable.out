stdout of test 'date` in directory 'modules/plain` itself:

#> Mtest:	other date
#> Mtest:	Mtimeout -timeout 420 ./date date
#> Mlog:	Mtimeout -timeout 210 Mserver -modulesilent -yaccsilent -d8 -db modules_plain date.mil
# Monet Database Server V4.1
# Copyright (c) 1993-1999, CWI & SION. All rights reserved.
# compiled for IRIX646.5/32bit on Fri Jan 14 01:04:33 MET 2000.
#script:date.mil

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.

#You are system administrator (adm)!
module(temporal);

(date("1998-10-10") < date("1998-11-09")).print;
[ true ]
(date("1998-10-10") < date("1998-09-11")).print;
[ false ]
(date("1998-10-10") < date("1998-10-09")).print;
[ false ]
(date("1998-10-10") < date("1998-10-11")).print;
[ true ]
(date("1998-10-10") < date("1998-10-10")).print;
[ false ]

(date("1998-10-10") > date("1998-11-09")).print;
[ false ]
(date("1998-10-10") > date("1998-09-11")).print;
[ true ]
(date("1998-10-10") > date("1998-10-09")).print;
[ true ]
(date("1998-10-10") > date("1998-10-11")).print;
[ false ]
(date("1998-10-10") > date("1998-10-10")).print;
[ false ]

(date("1998-10-10") >= date("1998-11-09")).print;
[ false ]
(date("1998-10-10") >= date("1998-09-11")).print;
[ true ]
(date("1998-10-10") >= date("1998-10-09")).print;
[ true ]
(date("1998-10-10") >= date("1998-10-11")).print;
[ false ]
(date("1998-10-10") >= date("1998-10-10")).print;
[ true ]

(date("1998-10-10") <= date("1998-11-09")).print;
[ true ]
(date("1998-10-10") <= date("1998-09-11")).print;
[ false ]
(date("1998-10-10") <= date("1998-10-09")).print;
[ false ]
(date("1998-10-10") <= date("1998-10-11")).print;
[ true ]
(date("1998-10-10") <= date("1998-10-10")).print;
[ true ]

(date("1998-10-10") = date("1998-11-09")).print;
[ false ]
(date("1998-10-10") = date("1998-09-11")).print;
[ false ]
(date("1998-10-10") = date("1998-10-09")).print;
[ false ]
(date("1998-10-10") = date("1998-10-11")).print;
[ false ]
(date("1998-10-10") = date("1998-10-10")).print;
[ true ]
